tools/analysis_tools/test_robustness.py

```diff
@@ -113,39 +33,25 @@ def parse_args():
             'spatter', 'saturate'
         ],
         help='corruptions')
+    parser.add_argument(
+        '--work-dir',
+        help='the directory to save the file containing evaluation metrics')
     parser.add_argument(
         '--severities',
         type=int,
         nargs='+',
         default=[0, 1, 2, 3, 4, 5],
         help='corruption severity levels')
-    parser.add_argument(
```

Comment:
https://mmdetection.readthedocs.io/en/3.x/user_guides/robustness_benchmarking.html should be updated

Reply:
'--eval' is remove now, so update the following command in https://mmdetection.readthedocs.io/en/3.x/user_guides/robustness_benchmarking.html

![image](https://user-images.githubusercontent.com/26483343/190392403-e2bddbc4-0100-4e5e-9f51-d611ebf93e25.png)